01-Fundamentos/Bloco-03-Introducao-a-HTML-e-CSS/3.01_Introducao_a_HTML_e_CSS/index.html

```diff
@@ -0,0 +1,34 @@
+
+    <img src="amigos.jpeg"  alt="" width="1000px">
+
+    <p id="ThiagoRamos">Thiago Ramos foi a primeira pessoa estudante que tive contato,<br>
+        foi ele que inclusive me ajudou a instalar o linux e suas extensões<br>
+        necessárias para o início do curso.</p>
+
+    <p id="WalaceBorges">Walace eu conheci através do Thiago, nós 3 trocamos ideia e discutimos<br>
+            os exercícios depois de feitos.</p>
+    
+
+   <a href="https://www.google.com.br/">Google</a>
```

Comment:
É bom quando o link abre outra aba ao invés de sair da página.
```suggestion
   <a href="https://www.google.com.br/" target="_blank">Google</a>
```